snuba/web/query.py

```diff
@@ -74,7 +75,11 @@ def parse_and_run_query(
     Runs a Snuba Query, then records the metadata about each split query that was run.
     """
     query_metadata = SnubaQueryMetadata(
-        request=request, dataset=get_dataset_name(dataset), timer=timer, query_list=[],
+        request=request,
+        dataset=get_dataset_name(dataset),
+        timer=timer,
+        query_list=[],
+        anonymized_request_body=format_query_anonymized(request.query).get_sql(),
```

Comment:
Technically the snuba language is different, But I guess we are ok with formatting the query as plain sql for now. We can make changes later and format it as a SnQL query. It will be tricky to rebuild the SnQL join anyway.